mkt/search/views.py

```diff
@@ -170,43 +32,29 @@ class SearchView(CORSMixin, MarketplaceView, GenericAPIView):
     authentication_classes = [RestSharedSecretAuthentication,
                               RestOAuthAuthentication]
     permission_classes = [AllowAny]
+    filter_backends = [SearchQueryFilter, PublicSearchFormFilter,
+                       ValidAppsFilter, DeviceTypeFilter, RegionFilter,
+                       ProfileFilter, SortingFilter]
     serializer_class = ESAppSerializer
     form_class = ApiSearchForm
     paginator_class = ESPaginator
 
-    def search(self, request):
-        """
-        Takes a request (expecting request.GET.q), and returns the serializer
-        and search query.
-        """
-        # Parse form.
-        form = self.form_class(request.GET if request else None)
-        if not form.is_valid():
-            raise form_errors(form)
-        form_data = form.cleaned_data
-
-        # Query and filter.
-        no_filter = (
-            request.GET.get('filtering', '1') == '0' and
-            acl.action_allowed(request, 'Feed', 'Curate'))
-        sq = WebappIndexer.get_app_filter(
-            request, search_form_to_es_fields(form_data), no_filter=no_filter)
-
-        # Sort.
-        sq = _sort_search(request, sq, form_data)
-
-        # Done.
-        page = self.paginate_queryset(sq)
-        return self.get_pagination_serializer(page), form_data.get('q', '')
-
     def get(self, request):
```

Comment:
Do you need a specific `get` method anymore or can you use DRF's `ListModelMixin` ? Also, can you drop `ReviewersSearchView.get` and just let it inherit from whatever the base `SearchView` class has ?